README.md

```diff
@@ -2,15 +2,142 @@
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Recognize the benefits of optional arguments
```

Comment:
Again, these 'recognize the benefit' LG seem highly unmeasurable.

Reply:
@sgharms makes sense.